apps/website-25/src/pages/_app.tsx

```diff
@@ -2,15 +2,50 @@ import '../globals.css';
 import type { AppProps } from 'next/app';
 import Head from 'next/head';
 import dynamic from 'next/dynamic';
+import posthog from 'posthog-js';
+/* eslint-disable import/no-extraneous-dependencies */
+import { PostHogProvider } from 'posthog-js/react';
 import {
   CookieBanner, Footer, isCurrentPath, Nav, constants,
 } from '@bluedot/ui';
 import clsx from 'clsx';
+import { useEffect } from 'react';
+import { Router } from 'next/router';
 import { Analytics } from '../components/Analytics';
 
+if (typeof window !== 'undefined') {
+  if (!process.env.NEXT_PUBLIC_POSTHOG_KEY) {
+    console.error('PostHog key is missing! Check your .env.local file');
+  } else {
+    console.log('Initializing PostHog...');
+    posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+      api_host: 'https://eu.i.posthog.com',
+      person_profiles: 'always',
+      // TODO: Do we want this to be "identified_only" or "always"?
+      loaded: (instance) => {
+        if (process.env.NEXT_PUBLIC_DEBUG_POSTHOG === 'true') instance.debug();
+        if (localStorage.getItem('cookies') === 'rejected') {
+          instance.opt_out_capturing();
```

Comment:
```suggestion
          posthog.set_config({ persistence: 'memory' });
```

We can still record analytics events, we just can't store data on the user's browser to track them between pages.